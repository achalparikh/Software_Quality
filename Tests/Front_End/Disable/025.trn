10_admin_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
07_John_Doe_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _