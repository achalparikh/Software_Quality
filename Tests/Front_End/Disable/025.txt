login
admin
disable
John Doe
00001
deposit
00001
500.00
logout