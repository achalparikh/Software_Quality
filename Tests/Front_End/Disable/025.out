Account 00001 is currently disabled and cannot perform any transactions.